activerecord/lib/active_record/counter_cache.rb

```diff
@@ -12,13 +12,21 @@ module ClassMethods
       #
       # * +id+ - The id of the object you wish to reset a counter on.
       # * +counters+ - One or more association counters to reset. Association name or counter name can be given.
+      # * <tt>:touch</tt> - Touch timestamp columns when updating.
+      #   Pass +true+ to touch +updated_at+ and/or +updated_on+. Pass a symbol to
+      #   touch that column or an array of symbols to touch just those ones.
       #
       # ==== Examples
       #
       #   # For Post with id #1 records reset the comments_count
       #   Post.reset_counters(1, :comments)
-      def reset_counters(id, *counters)
+      #
+      #   # For Post with id #1 records reset the comments_count
+      #   # and updates the `timestamp_attributes_for_update_in_model` attribute.
```

Comment:
`timestamp_attributes_for_updates_in_model` is private, so I'd rather we said `+updated_at+ and/or +updated_on+` like elsewhere 😁